_translator-files/po/de/Installation-for-Linux.po

```diff
@@ -144,10 +141,8 @@ msgstr "### Konfiguriere Jack mit QjackCtl"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:45
-#, fuzzy
-#| msgid "Jamulus clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`."
 msgid "Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`."
-msgstr "Jamulus Clients brauchen [JACK](https://jackaudio.org/). Du musst JACK aber zuerst einrichten. Am Besten machst du das mit `QjackCtl`."
+msgstr "Jamulus-Clients benötigen [JACK](https://jackaudio.org/) für den Betrieb, aber das musst du zuerst konfigurieren. Die empfohlene Methode ist die Verwendung von `QjackCtl`."
```

Comment:
Still prefering the old translation, although it's not that close to the english original.

Reply:
Correct to former translation
`Jamulus Clients brauchen [JACK](https://jackaudio.org/). Du musst JACK aber zuerst einrichten. Am Besten machst du das mit `QjackCtl`.`